client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -68,6 +67,7 @@ const LoggedOutForm = React.createClass( {
 		const { userData } = this.props.jetpackConnectAuthorize;
 		return (
 			<div>
+				{ this.props.renderFormHeader() }
```

Comment:
I'm not sure if passing a method from another class as a prop to be executed is a good idea... I mean, not even the parent instance, but a method as if it were a static one (which it's not, btw!).